tools/distribution/dogfood.py

```diff
@@ -27,6 +27,7 @@ def dogfood(dry_run: bool, repository_url: str, repository_name: str, repository
     dd_sdk_package_path = '../..'
     os.system(f'swift package --package-path {dd_sdk_package_path} resolve')
     dd_sdk_ios_package = PackageResolvedFile(path=f'{dd_sdk_package_path}/Package.resolved')
+    dd_sdk_ios_package.print()
```

Comment:
Is this print supposed to stay or is it for debug purposes?

Reply:
It is on purpose 👍 - this log was crucial to understand the problem and without it next CI logs may be too vague.